### PR TITLE
Re-pipes metastation

### PR DESCRIPTION
**Pull request by S34NW**
## What Does This PR Do
The entire atmos system on metastation was superbly broken, including plasma siphon vents (normally used in the plasma storage in atmos) being in like 7 different places on-station, and spaghetti pipes all over the place.
This PR standardises all non-special vents and scrubbers to the `\on` subtype.
It also re-pipes the entire map for more logical and less spaghetti.
This also tidies up nearly 3000 un-needed (same as default) lines of map var edits on random stuff that serve no purpose but bloat.

Things that might be considered balance changes, done for the sake of standardisation:
Viro scrubbers is connected to station waste, viro vents are still isolated with their own air supply.
Xenobio is now also connected to both station waste and distro.
AI Sat is now connected to station waste (in addition to the existing distro connection).

## Why It's Good For The Game
Having an actually functioning atmos system might help players breathe when we finally get map rotation.

Also, having a pipe system that looks like this:
image
Is bad for engineers mental health. 

## Images of changes
Var edits across the entire map, alongside an entire re-piping of the messy atmos net, look in mapdiffbot and cry along with me. 
image

## Changelog
:cl:
fix: Re-pipes metastation
/:cl:

### CODE REVIEW

**Comment by dearmochi** [comment]
image

Amazing

**Comment by S34NW** [comment]
Quite frankly, metastation needs re-piping, its pipenet is atrocious.

Examples of bad pipes:

- Why are these scrubbers on different y coordinates? image

- This loop literally does nothing at all.
image

- Who the hell designed this and thought it was okay?
image

**Review by Kyep — COMMENTED** [review]
Please include images showing what you changed.
Map diff bot images are not sufficient for this PR, because its diff files exceed max content length and the raw versions would be excessively tedious to review.

**Comment by S34NW** [comment]
> 
> 
> Please include images showing what you changed.
> Map diff bot images are not sufficient for this PR, because its diff files exceed max content length and the raw versions would be excessively tedious to review.

Its literally 75% of all scrubbers and vents on the station, apart from a screenshot of the entire map I don't know what to suggest.

**Comment by AffectedArc07** [comment]
For context, the "raw" images on MDB are the exact same as the embedded ones, its just a link directly to the image to get past github's field length limits.

TLDR: Its the same picture

**Comment by AffectedArc07** [comment]
Accidental vent placed in arrivals
image

**Review by Kyep — APPROVED** [review]

**Review by AffectedArc07 — APPROVED** [review]
This is queue jumping due to being a conflict bomb waiting to happen
